Plugins/DD4hep/src/DD4hepVolumeBuilder.cpp

```diff
@@ -7,6 +7,9 @@
 // file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 #include "Acts/Plugins/DD4hep/DD4hepVolumeBuilder.hpp"
+
+#include <boost/algorithm/string.hpp>
```

Comment:
Is this include still needed? I think the only usage of `boost::iequals` is removed in this PR.

Reply:
Done